dotnet/ios/ios13/AdoptingIOSDarkMode/AdoptingIOSDarkMode/After Adoption (2)/After2MoreViewController.cs

```diff
@@ -0,0 +1,154 @@
+﻿/*
+See LICENSE folder for this sample’s licensing information.
+
+Abstract:
+A view controller demonstrating typical creation of views: images, labels, blur, and vibrancy.
+ This is the code after dark mode adoption has happened.
+*/
+
+namespace AdoptingIOSDarkMode;
+public partial class After2MoreViewController : UIViewController {
+	public After2MoreViewController (IntPtr handle) : base (handle)
+	{
+	}
+
+	public override void LoadView ()
+	{
+		// Create our view controller's view.
+
+		View = new UIView {
+			TranslatesAutoresizingMaskIntoConstraints = false,
+			// DARK MODE ADOPTION: Changed from .white to a semantic color.
+			BackgroundColor = UIColor.SystemBackgroundColor
+		};
+
+		// Add various subviews, from top to bottom:
+		// - Star image (our app's logo)
+		// - Title label
+		// - Background image view
+		// - Visual effect view blurring that background image
+		// - Visual effect view for vibrancy
+		// - Vibrant label
+
+		var starImageView = ConfigureStarImageView ();
+		View.AddSubview (starImageView);
+
+		var titleLabel = ConfigureTitleLabel ();
+		View.AddSubview (titleLabel);
+
+		var backgroundImageView = ConfigureBackgroundImageView ();
+		View.AddSubview (backgroundImageView);
+
+		// DARK MODE ADOPTION: Changed from style: .light to an iOS 13
+		// blur style, .systemThinMaterial, that has different appearances in light
+		// and dark modes.
+		var blurEffect = UIBlurEffect.FromStyle (UIBlurEffectStyle.SystemThinMaterial);
+		var blurView = ConfigureBlurView (blurEffect);
+		View.AddSubview (blurView);
+
+		var vibrancyView = ConfigureVibrancyView (blurEffect);
+		blurView.ContentView.AddSubview (vibrancyView);
+
+		var vibrantLabel = ConfigureVibrantLabel ();
+		vibrancyView.ContentView.AddSubview (vibrantLabel);
+
+		// Add constraints to put everything in the right place.
+		SetupConstraints ();
+	}
+
+	UIImageView? starImageView;
+	UIImageView ConfigureStarImageView ()
+	{
+		starImageView = new UIImageView (UIImage.FromBundle ("StarImage"));
+		starImageView.TranslatesAutoresizingMaskIntoConstraints = false;
+		// DARK MODE ADOPTION: Changed to a color, from the "After" asset catalog,
+		// which has light and dark variants:
+		starImageView.TintColor = UIColor.FromName ("LightAndDarkHeaderColor");
+		return starImageView;
+	}
```

Comment:
The entire thing can be re-written in a more modern way:
```suggestion
	UIImageView starImageView = new UIImageView (UIImage.FromBundle ("StarImage")) {
		    TranslatesAutoresizingMaskIntoConstraints = false;
		    // DARK MODE ADOPTION: Changed to a color, from the "After" asset catalog,
		    // which has light and dark variants:
		    TintColor = UIColor.FromName ("LightAndDarkHeaderColor");
	        }
```

Doing the above is going to remove A LOT of the null check. If I am not mistaken, you added the private variable to the nullable because it was not initialised, that cascaded in a collection of null checks needed everywhere when in reality they are not. The suggestion will make your life easier and code cleaner. In the old times, you could not use the property initalizers which resulted in that 2 step init which is ugly ugly.